cmd/eval.go

```diff
@@ -209,9 +209,18 @@ func evalFn(cmd *cobra.Command, args []string) {
 		utils.Must(err)
 		printResults(results)
 		printLosses(results, losses)
-		search.WriteResults(reportPath, evalSet, results)
-		_, err = search.WriteResultsByExpectation(flatReportPath, evalSet, results)
-		utils.Must(err)
+		if _, ferr := os.Stat(reportPath); os.IsNotExist(ferr) {
+			log.Warnf("Cannot write results: reportPath ('%s') is not found!", reportPath)
```

Comment:
Why do you check stat here?
What are you trying to solve?

Reply:
Why file has to exist?

Reply:
You are writing the report, it might not exist yet, right?

Reply:
changed to check that this parameters are defined (without the checking we may encounter panic)